libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -596,12 +599,15 @@ export class DtStackedSeriesChart implements OnDestroy, OnInit {
             0.6 +
           1.5,
         relative:
-          (this._axisTicks.slice(-1)[0].valueRelative.toString().length + 1) *
+          ((this._axisTicks.slice(-1)[0].valueRelative * 100).toString()
```

Comment:
Not sure what this code is for. Can you help us out there? Seems pretty random with all these numbers :)

Reply:
Hi @thomaspink, it's a way to fix the second case (when `displayMode = percent`, only the last characters are displayed `0 %`, instead of the whole number `100 %`, `75 %`, `50 %`, ...). So I propose the adaptation of this line to consider the extra width the Y axis need: `this._axisTicks.slice(-1)[0].valueRelative` returns the highest relative value, a number between 0 and 1. In order to compute the needed width for displaying the percentage numbers I multiply this relative value length by 100.

Reply:
@tonidalmases could you add some comments in code please :)